cpu/lpc11u34/periph/spi.c

```diff
@@ -80,6 +80,7 @@ int spi_init_master(spi_t dev, spi_conf_t conf, spi_speed_t speed)
         case SPI_SPEED_1MHZ:
             spi->CR0 |= (11 << 8);
             break;
+        case SPI_SPEED_4MHZ:
```

Comment:
Ok.  But I can only do what I'm told to do, as I don't know the exact motivation behind this strange 4/5 Mhz... I guess it would be sane to allow only 4Mhz.